renderer/src/components/UpdateBanner.tsx

```diff
@@ -5,19 +5,19 @@ import { openReleaseNotes, restartToUpdate } from '../lib/station-config'
 const UpdateBanner = () => {
   const [isUpdateAvailable, setIsUpdateAvailable] = useState<boolean>(false)
 
-  const reload = async () => {
-    const updateStatus = await window.electron.getUpdaterStatus()
-    setIsUpdateAvailable(updateStatus.updateAvailable)
-  }
-
   useEffect(() => {
+    const reload = async () => {
+      const updateStatus = await window.electron.getUpdaterStatus()
+      setIsUpdateAvailable(updateStatus.updateAvailable)
+    }
+
     reload()
     const unsubscribeUpdateNotification = window.electron.stationEvents.onUpdateAvailable(() => {
       setIsUpdateAvailable(true)
     })
 
     return unsubscribeUpdateNotification
-  }, [])
+  }, [isUpdateAvailable])
```

Comment:
We are not using `isUpdateAvailable` anywhere inside the effect, are we?

We use `setIsUpdateAvailable` only, which we don't need to list in the dependencies, as state setters are guaranteed never to change. (In React parlance:
The identity of the `setIsUpdateAvailable` function is guaranteed to be stable so it’s safe to omit.)

Reply:
5cf1f28